docs/index.html

```diff
@@ -1,8 +1,141 @@
-<center>
-<img src="img/u-root-logo.png" alt="u-root logo" width=300 />
-</center>
+<!DOCTYPE html>
```

Comment:
Is the index.html still generated from the index.md without any modifications?

Reply:
Yes and now. I included the project description from the root README because the one here in the index.md is outdared and no longer accurate by now. See the last line in `rebuild.sh` that just cats the header that I took from index.md, then the description from the README at the root, and then the rest of index.md, piping the whole thing through pandoc as it had been done before.